com.fugro.roames.displaz.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "com.fugro.roames.displaz",
+  "branch": "master",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "displaz",
+  "tags": [
+    "master"
+  ],
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "displaz",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/nigels-com/displaz.git",
+          "branch": "flatpak"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "mkdir -p thirdparty/external/downloads",
+            "sed -i '/#    include <GL\\/glu.h>/d' thirdparty/glew/GL/glew.h"
+          ]
+        },
+        {
+          "type": "file",
+          "url": "http://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.0.tar.gz",
+          "dest-filename": "thirdparty/external/downloads/ilmbase-2.2.0.tar.gz",
+          "sha256": "ecf815b60695555c1fbc73679e84c7c9902f4e8faa6e8000d2f905b8b86cedc7"
+        }
+      ],
+      "build-commands": [
+        "mkdir build_external",
+        "pushd build_external && cmake ../thirdparty/external && make && popd",
+        "mkdir build",
+        "pushd build && cmake -DCMAKE_INSTALL_PREFIX=/app .. && make install && popd",
+        "mkdir -p /app/share/icons/hicolor/256x256/apps/",
+        "mkdir -p /app/share/applications/",
+        "mkdir -p /app/share/metainfo/",
+        "cp src/resource/displaz_icon_256.png            /app/share/icons/hicolor/256x256/apps/com.fugro.roames.displaz.png",
+        "cp flatpak/com.fugro.roames.displaz.desktop     /app/share/applications/com.fugro.roames.displaz.desktop",
+        "cp flatpak/com.fugro.roames.displaz.appdata.xml /app/share/metainfo/com.fugro.roames.displaz.appdata.xml"
+      ]
+    }
+  ]
+}
```

Comment:
Although this is not required, it would be nice if you put the newline char here.